src/aedat.hpp

```diff
@@ -17,10 +17,10 @@ struct AEDAT {
 
   struct PolarityEvent {
     uint32_t valid : 1;
-    uint32_t polarity : 1;
-    uint32_t x : 15;
-    uint32_t y : 15;
-    uint32_t timestamp : 32;
+    bool polarity : 1;
+    uint16_t x : 15;
+    uint16_t y : 15;
+    uint64_t timestamp : 32;
   } __attribute__((packed));
 
```

Comment:
that is not how bitfields work.

Reply:
Do you mean the type declaration? I unified it to `uint32_t`